diff.go

```diff
@@ -53,12 +53,25 @@ const (
 	// DiffInfoComponentExposeUpdated
 	DiffInfoComponentExposeUpdated DiffType = "component-expose-updated"
 
-	// DiffInfoComponentScaleUpdated
-	DiffInfoComponentScaleUpdated DiffType = "component-scale-updated"
+	// scale
+
+	// DiffInfoComponentScalePlacementUpdated
+	DiffInfoComponentScalePlacementUpdated DiffType = "component-scale-placement-updated"
+
+	// DiffInfoComponentScaleUp
+	DiffInfoComponentScaleUp DiffType = "component-scale-up"
```

Comment:
Why is there a `component-max-updated` and not a `component-min-updated`?

PS. I think both should be named `component-scale-mXX-updated`